android/beagle/src/main/java/br/com/zup/beagle/android/view/viewmodel/BeagleViewModel.kt

```diff
@@ -39,17 +41,18 @@ sealed class ViewState {
 }
 
 internal class BeagleViewModel(
+    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO,
```

Comment:
What do you think to use this instead?
```suggestion
    private val ioDispatcher: CoroutineDispatcher = CoroutineDispatchers.IO,
```

Reply:
I will change